### ISSUE TITLE
AVT 1 - React MultiSelect default has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4008]
Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP IBM Accessibility July 2019 Ruleset

Detailed Description
Go to the React MultiSelect w/initial selected item
Run DAP with the drop-down menu expanded and the following issues are found (see screenshot):
<img width="1168" alt="Screen Shot 2019-09-16 at 3 54 39 PM" src="https://user-images.githubusercontent.com/21676914/64992925-80dc5f80-d89a-11e9-91a0-1593ffb3f1ce.png">
